analytic_engine/src/instance/flush_compaction.rs

```diff
@@ -356,8 +332,11 @@ impl FlushTask {
             .set_last_flush_time(time::current_time_millis());
 
         info!(
-            "Instance flush memtables done, table:{}, table_id:{}, request_id:{}",
-            self.table_data.name, self.table_data.id, request_id
+            "Instance flush memtables done, table:{}, table_id:{}, request_id:{}, cost:{}",
```

Comment:
```suggestion
            "Instance flush memtables done, table:{}, table_id:{}, request_id:{}, cost_ms:{}",
```

Reply:
I guess it is better to do like this:
```suggestion
            "Instance flush memtables done, table:{}, table_id:{}, request_id:{}, cost:{}ms",
```